SlicerCART/src/SlicerCART.py

```diff
@@ -2763,7 +2859,6 @@ def disablePauseTimerButton(self):
 
   def toggleStartTimerButton(self):
       if (self.ui.SlicerDirectoryListView.count > 0):
-        if self.ui.StartTimerButton.isChecked():
             self.startTimer()
```

Comment:
previously, there was if self.ui.StartTimerButton.isChecked(). Reason to have removed it?

Reply:
We want to be able to start the timer with buttons of other functionalities including `paint`, `lasso paint`, `erase`, etc. It doesn't limit the start of the timer to the `start button itself`

Reply:
ok great! is it possible to write it in comment under the functon using '''' this functions ... ''' ?